atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/encoding/TicketsCellEncodingStrategy.java

```diff
@@ -111,4 +115,57 @@ public Stream<byte[]> getRowSetCoveringTimestampRange(long fromInclusive, long t
                 toInclusive + rowsPerQuantum - ((toInclusive % partitioningQuantum) % rowsPerQuantum) - 1);
         return LongStream.rangeClosed(startRow, endRow).mapToObj(TicketsCellEncodingStrategy::rowToBytes);
     }
+
+    public CellRangeQuery getRangeQueryCoveringTimestampRange(long fromInclusive, long toInclusive) {
+        long startPartition = getPartition(fromInclusive);
+        long endPartition = getPartition(fromInclusive);
+        if (startPartition == endPartition) {
+            return getQueryForSingleDoublyBoundedPartition(startPartition, fromInclusive, toInclusive);
+        }
+        throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
definitely log an error here

Reply:
probably should implement the naive approach (just run gQFSDBP on each partition and merge the results).